src/utils/strings.js

```diff
@@ -1 +1,3 @@
 export const deleteWhiteSpaces = str => str.replace(/\s/g, '');
+
+export const getFirstLetterFrom = value => value.slice(0, 1).toUpperCase();
```

Comment:
can we move the uppercase logic to the style? Applying `textTransform:'uppercase'` to the text. i've read it's more performant

Reply:
Sure!